### PR TITLE
Fixes potential null condition

### LINKED ISSUE #13648: Potential null conditional
### Description

```
TypeError: client.isValidPassPhrase is not a function
    at checkSeed (/Users/nejczdovc/Work/OpenSource/browser-laptop/app/browser/api/ledger.js:202:28)
    at Object.paymentPresent (/Users/nejczdovc/Work/OpenSource/browser-laptop/app/browser/api/ledger.js:184:13)
    at ledgerReducer (/Users/nejczdovc/Work/OpenSource/browser-laptop/app/browser/reducers/ledgerReducer.js:253:27)
    at reducers.reduce (/Users/nejczdovc/Work/OpenSource/browser-laptop/js/stores/appStore.js:191:24)
    at Array.reduce (<anonymous>)
    at applyReducers (/Users/nejczdovc/Work/OpenSource/browser-laptop/js/stores/appStore.js:189:68)
    at handleAppAction (/Users/nejczdovc/Work/OpenSource/browser-laptop/js/stores/appStore.js:253:16)
    at callbacks.forEach (/Users/nejczdovc/Work/OpenSource/browser-laptop/js/dispatcher/appDispatcher.js:107:7)
    at Array.forEach (<anonymous>)
    at AppDispatcher.dispatchToOwnRegisteredCallbacks (/Users/nejczdovc/Work/OpenSource/browser-laptop/js/dispatcher/appDispatcher.js:106:20)
    at AppDispatcher.dispatchInternal (/Users/nejczdovc/Work/OpenSource/browser-laptop/js/dispatcher/appDispatcher.js:132:10)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
```

You could get error if bat-client is not initialised yet.

### PR DESCRIPTION
**Pull request by NejcZdovc**
Resolves #13648
## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).
- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review). (Ask a Brave employee to help if you cannot access this document.)

## Reviewer Checklist:

- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review) if one was not already requested.

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: app/browser/api/ledger.js [hunk 1]
```diff
@@ -193,8 +193,13 @@ const paymentPresent = (state, tabId, present) => {
...
 const checkSeed = (state) => {
   const seed = ledgerState.getInfoProp(state, 'passphrase')
+  let localClient = client
 
-  if (seed && !client.isValidPassPhrase(seed)) {
+  if (!localClient) {
+    localClient = require('bat-client')
+  }
+
+  if (seed && typeof localClient === 'object' && !localClient.isValidPassPhrase(seed)) {
```
Comment (bsclifton) [inline app/browser/api/ledger.js]: I don't think the check `typeof localClient === 'object'` is needed here. Even if it's null, typeof will return `object`

Comment (NejcZdovc) [reply app/browser/api/ledger.js]: then maybe just plain and simple `localClient`

Comment (bsclifton) [reply app/browser/api/ledger.js]: That would be perfect!

Comment (NejcZdovc) [reply app/browser/api/ledger.js]: done

**Comment by bsclifton** [comment]
master ef3ef0f0c08a514bc14559dfe3a8e13a1b65439a
0.23.x f585f37fb26e1835746cc1b7a46e6e9b6165a955
0.22.x 3c27b1dc25d2fc72fd219166112aa0408aa7993e

**Pull request merged**